content/_index.md

```diff
@@ -12,6 +12,9 @@ weight: -10
 
 The group of JProf. Volkamer is working on novel methods for in-silico toxicology predictions.
 
+> Breaking news: join us in our [collective effort](https://github.com/volkamelrab/covid19-SBapproach) to understand Covid-19 better
```

Comment:
hi Jaime, nice work :)
I think you have a little type here :https://github.com/volkamelrab/covid19-SBapproach
```suggestion
> Breaking news: join us in our [collective effort](https://github.com/volkamerlab/covid19-SBapproach) to understand Covid-19 better
```